package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@apiaryio/metamorphoses",
-  "version": "0.10.0-rc.1",
+  "version": "0.12.0-rc.0",
```

Comment:
with the `blueprint-markdown-renderer@1.0.0` I'd say this `rc-x` can go away, let's release this metamorphoses as `0.12.0` `stable` :-)